bentoml/cli/aws_ec2.py

```diff
@@ -0,0 +1,381 @@
+            error_code, error_message = status_pb_to_error_code_and_message(
+                get_deployment_result.status
+            )
+            raise CLIException(f"{error_code}:{error_message}")
+
+        delete_deployment_result = yatai_client.deployment.delete(
+            namespace=namespace, deployment_name=name, force_delete=force
+        )
+        if (
+            delete_deployment_result.status.status_code
+            != yatai_proto.status_pb2.Status.OK
+        ):
+            error_code, error_message = status_pb_to_error_code_and_message(
+                get_deployment_result.status
+            )
+            raise CLIException(f"{error_code}:{error_message}")
+
+        _echo(f"Successfiully deleted AWS EC2 deployment '{name}'", CLI_COLOR_SUCCESS)
+
+    @aws_ec2.command(help="Get EC2 deployment")
+    @click.argument("name", type=click.STRING)
+    @click.option(
+        "-n",
+        "--namespace",
+        type=click.STRING,
+        help='Deployment namespace managed by BentoML, default value is "dev" which '
+        'can be changed in BentoML configuration yatai_service/default_namespace',
+    )
+    @click.option(  # pylint: disable=unused-variable
+        "-o", "--output", type=click.Choice(["json", "yaml", "table"]), default="json"
+    )
+    def get(name, namespace, output):
+        yatai_client = get_default_yatai_client()
+        describe_result = yatai_client.deployment.describe(namespace, name)
+
+        if describe_result.status.status_code != yatai_proto.status_pb2.Status.OK:
+            error_code, error_message = status_pb_to_error_code_and_message(
+                describe_result.status
+            )
+            raise CLIException(f"{error_code}:{error_message}")
+
+        get_result = yatai_client.deployment.get(namespace, name)
+        if get_result.status.status_code != yatai_proto.status_pb2.Status.OK:
+            error_code, error_message = status_pb_to_error_code_and_message(
+                describe_result.status
+            )
+            raise CLIException(f"{error_code}:{error_message}")
+
+        _print_deployment_info(get_result.deployment, output)
+
+    @aws_ec2.command(help="Update existing AWS EC2 deployments")
+    @click.argument("name", type=click.STRING)
+    @click.option(
+        "-b",
+        "--bento",
+        "--bento-service-bundle",
+        type=click.STRING,
+        callback=parse_bento_tag_callback,
+        help="Target BentoService to be deployed, referenced by its name and version "
+        'in format of name:version. For example: "iris_classifier:v1.2.0"',
+    )
+    @click.option(
+        "-n",
+        "--namespace",
+        type=click.STRING,
+        help='Deployment namespace managed by BentoML, default value is "dev" which '
+        'can be changed in BentoML configuration yatai_service/default_namespace',
+    )
+    @click.option(
+        "--region", type=click.STRING, help="Region to deploy service in",
+    )
```

Comment:
I am not sure we can change region after cloudformation deployment, can you verify this?

Reply:
Yes you are right,we cant change region of stack.

Reply:
fixed